src/main/webapp/app/exercises/shared/plagiarism/plagiarism-run-details/plagiarism-run-details.component.ts

```diff
@@ -61,14 +61,44 @@ export class PlagiarismRunDetailsComponent extends PlagiarismAndTutorEffortDirec
     updateChartDataSet(data: number[]) {
         let ngxDataEntity;
         this.ngxData = [];
-        data.forEach((value, position) => {
+        this.totalDetectedPlagiarisms = 0;
+
+        // TextBased Exercises return 100 steps in data, Modeling only 10
```

Comment:
I think we should fix this directly. Did you spot where this comes from?

Reply:
Yes, I found it and fixed it. Locally it works now, but couldn't test it on the test servers myself